src/main/scala/interplay/PlayBuildBase.scala

```diff
@@ -44,6 +32,13 @@ object PlayBuildBase extends AutoPlugin {
     val playCrossReleasePlugins = settingKey[Boolean]("Whether the sbt plugins should be cross released or not")
     val playBuildRepoName = settingKey[String]("The name of the repository in the playframework GitHub organization")
 
+    val playBuildScala210Version = settingKey[String]("The version of Scala 2.10 to use")
```

Comment:
I'd hoped this would give a 'base' definition for a Scala version that when you changed it would flow through to other settings. But I realise that I don't know exactly how settings work - especially on multi-project cross-built environments - so I'll just revert this. :)